mozci/task.py

```diff
@@ -253,6 +253,10 @@ def _load_errorsummary(self):
             #     adr.config["cache"]["retention"],
             # )
 
+    @property
+    def configuration(self):
+        return self.label[len("test-") : self.label.index("/")]
```

Comment:
I wonder if there is a better way to get this

Reply:
Yeah, I'm not sure if this will work since certain things (like `-fis`) appear after the `/`. The build type (debug/opt) also comes after the slash. I'm not sure the best way of extracting this.. Maybe a regex?

To do it properly likely involves loading the actual task definition from the taskgraph and inspecting attributes, but that sounds expensive.

Reply:
I'm kind of assuming you care about `build_type` and `unittest_variants` (aka fis, spi, etc) here.

Reply:
Yeah, I care about any alternative platform/config/whatever where a group could run

Reply:
Maybe the best way could be to get the full name and remove suite names and numbers?

E.g. `test-linux1804-64-qr/debug-mochitest-plain-fis-e10s-7` -> `test-linux1804-64-qr/debug-*-fis-e10s-*`

Reply:
Another option would be to get those attributes as part of the AD queries that we already do, if the data is available in the Treeherder or Unittest table (e.g. in unittest I see build.type and run.type could be useful).

In summary, I need a way to have something that I will return in the bugbug reply and that the taskgraph can easily use to select the platform/configuration.

Reply:
Yeah removing the suite from the label seems like a good bet for now. But having this "unique config" value easily accessible seems useful for other things. It might be worth investigating storing it somewhere accessible in the taskgraph such that it gets picked up by AD.